README.md

```diff
@@ -193,7 +193,7 @@ response = client.get_recording(RECORDING_URL)
 response = client.start_verification(number='441632960960', brand='MyApp')
 
 if response['status'] == '0':
-  print 'Started verification request_id=' + response['request_id']
+  print 'Started verification request_id={}'.format(response['request_id'])
```

Comment:
In general, I like the move to `format`, but I feel like these are currently less readable than the existing string concatenation. I'd be really happy to accept this PR if you could switch to named placeholders, which I feel would be more readable than the existing code, e.g.:
```python
print 'Started verification request_id={request_id}'.format(request_id=response['request_id'])
```

Reply:
@judy2k,
I have formatted files with named placeholders.
Please check.